runtime/src/lib.rs

```diff
@@ -702,7 +713,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(InitBabe,),
+	(MigrateBags,),
```

Comment:
I'm not sure what these changes do. IDK what a "bag" is in the context of Substrate.